p2p/src/main/java/bisq/network/p2p/P2PService.java

```diff
@@ -311,6 +311,7 @@ private void onNetworkReady() {
                 "seedNodeOfPreliminaryDataRequest must be present");
 
         requestDataManager.requestUpdateData();
+        UserThread.runPeriodically(() -> requestDataManager.requestUpdateData(), 1, TimeUnit.HOURS);
```

Comment:
Maybe we should make that configurable?

Reply:
agreed. However, would you (for starters) really offer the user the change to change that?

Reply:
not really for users, more for seed node operators. but yes not really needed, can be added later as well once needed...